iwyu_include_picker.cc

```diff
@@ -621,6 +621,28 @@ const char* stdlib_cpp_public_headers[] = {
   "<version>",
 };
 
+// Common public -> public include mappings for C++ standard library
```

Comment:
Once you've taken care of https://github.com/include-what-you-use/include-what-you-use/pull/1309#issuecomment-1753583267 -- please also add a note here to keep the external mapping in sync à la https://github.com/include-what-you-use/include-what-you-use/blob/master/iwyu_include_picker.cc#L626.

And a corresponding note in the new .imp file.